src/main/java/xyz/vegaone/easytrackingv2/repo/OrganizationRepo.java

```diff
@@ -1,7 +1,13 @@
 package xyz.vegaone.easytrackingv2.repo;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import xyz.vegaone.easytrackingv2.domain.OrganizationEntity;
 
+import java.util.List;
+
 public interface OrganizationRepo extends JpaRepository<OrganizationEntity, Long> {
+
+    @Query(value = "SELECT new xyz.vegaone.easytrackingv2.domain.OrganizationEntity(name, id) FROM OrganizationEntity")
```

Comment:
I believe this method is not needed since the `JpaRepository` already has a method called `findAll()`. It should fetch a list of all the organizations. Could you please look into it?